question-service/model/question-orm.js

```diff
@@ -41,11 +41,11 @@ export async function ormUpdateQuestion(Question, changes) {
     }
 }
 
-export async function ormFindQuestionRandomly(difficulty, title=null) {
+export async function ormFindQuestionRandomly(difficulty, title) {
     try {
         const params = {};
         if (title) {
-            params['QuestionTitle'] = {$ne : title};
+            params['QuestionTitle'] = title;
```

Comment:
Why was this changed? This needs to be in place in order to ensure that when generating new questions, the current question does not appear again. While I understand that it might be for testing purposes, it should not be left in the PR.

Reply:
@ureshiiYing  I see! Thanks for pointing it out. I misunderstood its purpose and removed it as it allowed my feature to work before & after testing. I will add it back.